tests/unit/__mocks__/helpersMock.ts

```diff
@@ -1,3 +1,12 @@
 export const prettifyNumber = (): string => {
   return "";
 };
+
+export const compareString = (stringOne: string, stringTwo: string) => {
+  const strings = [stringOne, stringTwo];
+  if (!strings.every(str => typeof str == "string"))
+    throw new Error(
+      `String one: ${stringOne} String two: ${stringTwo} one of them are falsy or not a string`
+    );
+  return stringOne.toLowerCase() == stringTwo.toLowerCase();
```

Comment:
Is this function being re-implemented? What's the go here?

Reply:
Ah I'll remove that because I left it in the middle of the resolving test issue.